server/controllers/finance/reports/debtors/openDebtors.handlebars

```diff
@@ -19,19 +20,19 @@
           <tr class="text-capitalize text-center" style="background-color: #ddd;">
             <th>{{translate "FORM.LABELS.REFERENCE"}}</th>
             <th>{{translate "FORM.LABELS.PATIENT"}}</th>
-            <th>{{translate "TABLE.COLUMNS.LAST_INVOICE"}}</th>
-            <th>{{translate "TABLE.COLUMNS.LAST_PAYMENT"}}</th>
+            {{#if details.showDetailedView}}<th>{{translate "TABLE.COLUMNS.LAST_INVOICE"}}</th>{{/if}}
+            {{#if details.showDetailedView}}<th>{{translate "TABLE.COLUMNS.LAST_PAYMENT"}}</th>{{/if}}
             <th class="text-center">{{translate "TABLE.COLUMNS.TOTAL_DEBT"}}</th>
           </tr>
         </thead>
         <tbody>
           {{#each debtors as | debtor |}}
             <tr>
-              <th>{{debtor.reference }}</th>
+              <td>{{debtor.reference}}</td>
               <td>{{debtor.display_name }}</td>
-              <td>{{date debtor.lastInvoiceDate}}</td>
-              <td>{{date debtor.lastPaymentDate}}</td>
-              <td class="text-right">{{currency debtor.debt ../metadata.enterprise.currency_id}}</td>
+              {{#if ../details.showDetailedView}}<td>{{date debtor.lastInvoiceDate}}</td>{{/if}}
```

Comment:
I think the SQL now uses `latestInvoiceDate` and `latestCashDate`. These references should either be updated or the SQL should be changed to match this interface. 

As it is now I believe this default to the current date of the report as it will be rendering a `new Date(null)`.